column_transforms/rasgo_cast/rasgo_cast.sql

```diff
@@ -0,0 +1,7 @@
+SELECT
+*
+{%- for col_expr, new_type in new_cols %}
```

Comment:
i like this idea but the nested arg types this requires isn't something we have supported in the UI yet... life may be easier here if you just create two lists and force them to be ordinally aligned so like:
column_list: ['col1', 'col2', 'col3']
type_cast: ['string', 'int', 'float']

the other option is to go with a more complex dict but that feels like overkill here. can discuss more live